book/proposal/proposal.md

```diff
@@ -0,0 +1,85 @@
+
+- Downstream communication issues.
+- Misunderstood requirements.
+- Incorrect implementations.
+- Developer frustration.
+- Scope change or creep. 
+- Lost context over time.
+- Code that does not properly convey intent.
+- Unintended consequence of poorly modeled data.
+- Incorrect prioritization.
+
+Our workflows and structures create communication barriers we must break down in order to move our implementations closer to their domains. As developers, our goals should include moving code closer to the vocabulary of the domain. This requires understanding not only the problem being solved by a feature but also, more broadly an understanding of the field we are working in.
+
+Design thinking offers a toolset for gathering insight into the domain that can not only help engineers understand a user's problem more fundamentally, it can also offer byproducts such as:
+
+- Providing a better understanding of the entire domain.
+- A clearer path to a domain language.
+- Understanding problem nuance.
+- Gaining lessons learned of the user.
+- Clear prioritization of tasks.
+- Indetifying focused solutions.
+- Providing insight into how an implementation may evolve.
+
+Following the path of a team replacing a legacy application with a new application this book will explore tools that can be utilized to break down a domain and better communicate with domain experts. The book will introduce a number of design thinking exercises and explain how they can be applied in different scenarios to help agile developers craft clearer implementations.
+
+## What this book will cover:
+
+- Introduce the concept of design thinking.
+- Introduce the following design thinking methodologies from the perspective of the developer:
+	- Stakeholder mapping.
+	- Interviewing.
+	- Job shadowing.
+	- Rose Bud Thorn.
+	- Affinity Mapping.
+	- Experience diagramming.
+	- Importance/Difficulty matrix.
+	- Feature Voting.
+	- Creative matrixes.
+	- Technical prototyping.
+	- Story boarding UI's
+	- Stakeholder validation.
+- Introduce domain driven design concepts:
+	- Ubiquitous language.
+	- Context bounding.
+	- Domain events.
+- Outline effective tools for using design thinking to create ubiqoutous language, bounded contexts and domain events.
+- Outline rules for effective design thinking on agile teams.
+- Discuss ideas for bringing these methods into teams that are not familiar with design thinking.
+
+## Audience:
+This book is suitable for developers, managers, product manager, project managers, designers and anyone looking for methods to bring implementation closer to a domain.
+
+## Why is your book different?
+
+Design thinking books are often aimed at hard D design. This book will focus instead on using design thinking tools to create a different sort of outcome then is 
```

Comment:
It is different, it really is.... But what to say what to say...

Reply:
:)

your perspectively is always uniquely yours. Start with your experience and how that experience led up to this way of thinking.

Reply:
.....your experience gives you a ton of credibility to your views....and your views on design thinking (especially the language/rosetta stone style concept) will prove extremely valuable to others....even those doing design thinking (or thinking they are designedly thinking :))